data/language/en-GB.txt

```diff
@@ -3585,7 +3585,7 @@ STR_6134    :Clear Scenery
 STR_6135    :Client sent invalid request
 STR_6136    :Server sent invalid request
 STR_6137    :OpenRCT2, a free and open source clone of Roller Coaster Tycoon 2.
-STR_6138    :OpenRCT2 is the work of many authors, a full list can be found in “contributors.md”. For more information, visit http://github.com/OpenRCT2/OpenRCT2
+STR_6138    :OpenRCT2 is the work of many authors, a full list can be found in “contributors.md”. For more information, visit http://github.com/OpenRCT2/OpenRCT2.
```

Comment:
Any reason for this change? This isn't really part of this PR. If you think it should be changed it would be best to address in a different PR.

Reply:
What I didn’t realize was that I could use the feature branch instead of directly merging. For our class, we essentially had to put our names in that string, so I just removed it. If there is a way to remove this part, I would be all for it.

Reply:
Just correct it in another commit. When we squash merge this it will be ignored.

Reply:
Ok, sweet.